parse.go

```diff
@@ -0,0 +1,53 @@
+package vagrantutil
+
+import (
+	"bufio"
+	"strings"
+)
+
+// parseData parses the given vagrant type field from the machine readable
+// output (records).
+func (v *Vagrant) parseData(records [][]string, typeName string) (string, error) {
```

Comment:
Maybe you should print the returned error instead of printing it in this function? It would make it more cleaner.